third_party/jpeg/jpeg.BUILD

```diff
@@ -5,6 +5,8 @@ licenses(["notice"])  # custom notice-style license, see LICENSE.md
 
 exports_files(["LICENSE.md"])
 
+load("@//third_party:common.bzl", "expand_header_template")
```

Comment:
I guess on that one, we do not really have an alternative option?
I think one thing we can do is to define these in either tensorflow/tensorflow.bzl, or move this file to tensorflow/common.bzl?